README.md

```diff
@@ -116,6 +116,9 @@ Pulumi provider repositories have the following general structure:
 
 ## Adding Mappings, Building the Provider and SDKs
 
+> [!NOTE]
+> Recent versions of this repository leverage [automatic token mapping](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/docs/automatic-token-mapping.md) as an alternative to mapping resources manually. Unless you have a specific use case that requires manual mapping, you may skip this section.
```

Comment:
I think technically, you still need the later steps in this section. Maybe try this instead:
```suggestion
> Recent versions of this repository leverage [automatic token mapping](https://github.com/pulumi/pulumi-terraform-bridge/blob/master/docs/automatic-token-mapping.md) as an alternative to mapping resources manually. If you did not see any warnings about unmapped resources in the step above, you may skip directly to building the provider binary.
```